views/api/userConfigurations/index.js

```diff
@@ -1,51 +1,21 @@
 import { apiRoutes } from '../../routes/routes'
-import { BASE_REQUEST_OPTIONS } from '../../../constants'
-import { handleApiResponse } from '../helpers'
+import http from '../http'
 
 const userConfigurations = {
-  all: async (userId) => {
-    const response = await fetch(
+  all: async (userId) =>
+    http.get(apiRoutes.configurations.userConfigurations(userId)),
+  create: async (userId, configAttributes) =>
+    http.post(
       apiRoutes.configurations.userConfigurations(userId),
-      BASE_REQUEST_OPTIONS
-    )
-
-    return handleApiResponse(response)
-  },
-  create: async (userId, configAttributes) => {
-    const response = await fetch(
-      apiRoutes.configurations.userConfigurations(userId),
-      {
-        ...BASE_REQUEST_OPTIONS,
-        method: 'POST',
-        body: JSON.stringify(configAttributes),
-      }
-    )
-
-    return handleApiResponse(response)
-  },
-  destroy: async (userId, configId) => {
-    const response = await fetch(
+      configAttributes
+    ),
+  destroy: async (userId, configId) =>
```

Comment:
maybe, I think the difference is confusing between destroy and delete. I think our convention is to call destroy, but the http method in the http module is delete because that is what the actual http method we're sending is.